### ISSUE TITLE
Multiple servers?

### ISSUE THREAD

**Issue by artpolikarpov** [RocketChat/meteor-streamer#8]
Does this works with multiple servers?

**Comment by ejfrancis** [comment]
Hey @artpolikarpov,

In PR #5 I added the option to provide your own DDP connection to another server. But I've just run into an issue today where you can't have multiple DDP connections *with the same name* because of [this on the client](https://github.com/RocketChat/meteor-streamer/blob/8dcf55872ca661a8502b77251ac21ff61942e4c5/packages/rocketchat-streamer/client/client.js#L48) and [this on the server](https://github.com/RocketChat/meteor-streamer/blob/33888441788369dabf9049973984beaf82ef1084/packages/rocketchat-streamer/server/server.js#L18). That check made sense when you could only use one DDP connection across all instances, but now that it supports multiple instances it's a very valid use case to have multiple connections with the same name to different servers. 

I need this feature so I think I'll get it up in a PR soon. @rodrigok having multiple streamers of the same name to multiple endpoints shouldn't be a problem right? I'll just have to make the "already exists" check a little more intelligent

**Comment by ejfrancis** [comment]
Hey @artpolikarpov,

In PR #5 I added the option to provide your own DDP connection to another server. But I've just run into an issue today where a warning is logged if you have multiple streamers *with the same name* because of [this on the client](https://github.com/RocketChat/meteor-streamer/blob/8dcf55872ca661a8502b77251ac21ff61942e4c5/packages/rocketchat-streamer/client/client.js#L48) and [this on the server](https://github.com/RocketChat/meteor-streamer/blob/33888441788369dabf9049973984beaf82ef1084/packages/rocketchat-streamer/server/server.js#L18). That check made sense when you could only use one DDP connection across all instances, but now that it supports multiple instances it's a very valid use case to have multiple connections with the same name to different servers. 

@rodrigok is it alright if I remove that warning if you're providing an instance with a different ddp connection?

**Comment by neurobe** [comment]
@ejfrancis  @rodrigok yes, can simplify code when switching modes of operation, like in my case switching from networked to standalone operation. look forward to update pls.

**Comment by neurobe** [comment]
@ejfrancis @rodrigok  opened my feature request related to this issue #9

**Comment by lorensr** [comment]
If the question does this package work if your Meteor server is scaled to multiple containers, I think the answer is no. Events aren't stored in a database—events received by container A are sent out to all subscriptions that came to container A, but not to subscriptions that came to container B.

**Comment by maxnowack** [comment]
@lorensr see #26 😊
